docs/SuccessFactors-connector.md

```diff
@@ -0,0 +1,23 @@
+# SuccessFactors Connection
```

Comment:
Add SAP before SuccessFactors. See how Olivia has Ariba topics: https://cloud.google.com/data-fusion/docs/how-to/connect-to-sap-ariba-batch-source